direct-auth-samples/src/main/resources/templates/register-password.html

```diff
@@ -51,13 +51,15 @@
 
 <body>
 
-<a href="/" class="btn btn-info pull-left">Index Page</a>
+<div style="padding:1em">
+    <a href="/"><i class="fa fa-home fa_custom fa"></i> Home</a>
+</div>
 
 <div class="changepassword-form">
     <form th:action="@{/register-password}" method="post">
-        <h2 class="text-center">Change Password</h2>
+        <h2 class="text-center"><span th:text="${title}"></span></h2>
```

Comment:
Should we update the page title too (on line 21)

Reply:
done